src/App.vue

```diff
@@ -203,4 +203,19 @@ th {
   border: 1px solid black;
   padding: 10px;
 }
+
+/* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
+form {
+  margin: auto;
+}
+
+form,
+button {
+  margin: 20px;
+}
```

Comment:
Overrides previous margin:auto in the form selector. To fix, remove the "form" in the "form, button" selector so that only buttons have margin:20px